{"name": "test_add[nonefail]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <testcase.test_calc.TestCalc object at 0x0000023BC139F160>, a = None, b = 9, expect = 0, get_calc = <test_code.calc.Calculator object at 0x0000023BC1308670>, caculate = None\n\n    @pytest.mark.parametrize(\"a,b,expect\", adddatas_data,\n                             ids=myid_add)\n    @pytest.mark.run(order=1)\n    def test_add(self, a, b, expect,get_calc,caculate):\n        # 实例化计算机类\n        # clac = Calculator()\n        if isinstance(a, str):\n            print(\"非数字类型\")\n            assert False\n    \n        elif a is None:\n            print(\"不能为空\")\n>           assert False\nE           assert False\n\ntest_calc.py:60: AssertionError"}, "attachments": [{"name": "stdout", "source": "05041052-6beb-4d64-8ea8-645dc0310f8f-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "a", "value": "None"}, {"name": "b", "value": "9"}, {"name": "expect", "value": "0"}], "start": 1612147341732, "stop": 1612147341733, "uuid": "d2c688a5-598c-4247-9db6-de6f32a6d74c", "historyId": "a0d5437bfa1d881271b117c223a33d88", "testCaseId": "a7b29b08300809261dc069b49da8543b", "fullName": "test_calc.TestCalc#test_add", "labels": [{"name": "tag", "value": "run(order=1)"}, {"name": "suite", "value": "test_calc"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "LAPTOP-CV4D84I5"}, {"name": "thread", "value": "32636-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc"}]}